From f96a6a34261ee3e87b9e208b81df918e7b11f039 Mon Sep 17 00:00:00 2001
From: "Ho, Yu Xin" <yu.xin.ho@intel.com>
Date: Tue, 18 Sep 2018 18:05:28 +0800
Subject: [PATCH] ASoC: Intel: Common: add ACPI match for GPMRB machine driver

Load GPMRB machine driver and ADSPFW.bin when ACPI ID INT34C3
is matched.

Signed-off-by: Ho, Yu Xin <yu.xin.ho@intel.com>
---
 sound/soc/intel/common/soc-acpi-intel-bxt-match.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/sound/soc/intel/common/soc-acpi-intel-bxt-match.c b/sound/soc/intel/common/soc-acpi-intel-bxt-match.c
index ce78e0d..874ec7e 100644
--- a/sound/soc/intel/common/soc-acpi-intel-bxt-match.c
+++ b/sound/soc/intel/common/soc-acpi-intel-bxt-match.c
@@ -93,6 +93,12 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_bxt_machines[] = {
 		.drv_name = "bxt_ivi_ull",
 		.fw_filename = "intel/dsp_fw_ull_bxtn.bin",
 	},
+#elif IS_ENABLED(CONFIG_SND_SOC_INTEL_BXT_GPMRB_MACH)
+	{
+		.id = "INT34C3",
+		.drv_name = "gpmrb_machine",
+		.fw_filename = "ADSPFW.bin",
+	},
 #else
 	{
 		.id = "INT34C3",
-- 
1.9.1

